src/App.tsx

```diff
@@ -15,36 +15,107 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphabetically = 'alphabeticaly',
```

Comment:
There is a typo in the `SortType.Alphabetically` value. It should be 'alphabetically' instead of 'alphabeticaly'. This typo could cause issues when trying to sort alphabetically.